lib/proofer/vendor/mock.rb

```diff
@@ -46,6 +50,26 @@ def submit_phone(phone_number, session_id = nil)
         end
       end
 
+      # rubocop:disable MethodLength
```

Comment:
`MethodLength` is one of the easiest offenses to fix. I would like to propose a policy that we should never disable this cop, and add it to our checklist for PR reviews. I would be happy to pair with you to show you how to fix this offense. At a basic level, this is done by splitting this up into smaller methods. For example:
```ruby
def submit_state_id(state_id_data, session_id = nil)
  return unsuccessful_state_confirmation if invalid_state_submission?(state_id_data)
  successful_state_confirmation
end

def invalid_state_submission?(state_id_data)
  state_not_supported?(state_id_data) || invalid_state_id_number?(state_id_data)
end

def state_not_supported?(state_id_data)
 !SUPPORTED_STATES.include? state_id_data[:state_id_jurisdiction]
end

etc.
```
By doing this, you start to see that you're passing around the `state_id_data` variable, which is an indication this probably fits better in a separate class that will only be responsible for validating the state data and returning the appropriate response.

Reply:
I agree with you about refactoring the `state_id_data` into a separate class, I'll create a new PR for that. It looks like all the other methods(`financials` etc.) might also use that type of refactoring.